src/connections/destinations/catalog/actions-fullstory-cloud/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: FullStory Cloud Mode (Actions)
+hide-boilerplate: true
+hide-dossier: false
+id: 62d9aa9899b06480f83e8a66
+---
+{% include content/plan-grid.md name="actions" %}
+
+[FullStory](https://www.fullstory.com/) lets product and support teams easily understand everything about the customer experience. The Segment integration for FullStory helps accurately identify your customers within FullStory.
+
+FullStory’s cloud mode Segment integration allows you send user and event data to FullStory from your servers and Cloud Apps so that you apply it to your analysis throughout FullStory. For example, you could build a funnel to analyze drop-off of users who engaged with a certain marketing campaign.
+
+If you want to use FullStory’s tagless autocapture, use the [FullStory Device Mode (Actions) web destination](https://segment.com/docs/connections/destinations/catalog/actions-fullstory/). However, if you want to capture custom user properties and events from other [server-side sources](https://segment.com/docs/connections/sources/#server) or [cloud apps](https://segment.com/docs/connections/sources/#cloud-apps), such as recurring subscription purchases, use this cloud mode destination.
+
+### Overview
+
+The FullStory cloud mode destination sends information about your users and related events to FullStory. It uses [FullStory’s REST APIs](https://developer.fullstory.com).
+
+- **Identify User:** Converts Segment [Identify](https://segment.com/docs/connections/spec/identify/) calls to [FullStory Set User Properties API calls](https://developer.fullstory.com/set-user-properties). Use this to set custom attributes which can be used to search and segment within FullStory.
+- **Track Custom Event**: Converts Segment [Track](https://segment.com/docs/connections/spec/track/) calls to FullStory custom event API calls. Use this to capture more context about your user’s experience on your site or to capture user’s actions in other applications to build a more complete understanding of your user’s overall experience.
```

Comment:
Let's link "FullStory custom event API" to this URL: https://developer.fullstory.com/server-events. I just confirmed with Clay Gillman that this will be the URL for the soon-to-be-published API docs (within the next few days).